.github/workflows/docker-release.yml

```diff
@@ -39,3 +39,36 @@ jobs:
       - name: Image digest
         run: |
           echo ${{ steps.docker_build.outputs.digest }}
+  notify-complete-success:
```

Comment:
@kridai If you think, we need notifications for RPM workflow also, Please let me know.

Reply:
yes I think we would need the notification in alert-dev-interfaces for failure cases.